cf-testd/cf-testd.c

```diff
@@ -118,52 +117,35 @@ CFTestD_Config *CFTestD_CheckOpts(int argc, char **argv)
     CFTestD_Config *config = CFTestD_ConfigInit();
     assert(config != NULL);
 
-    while ((c = getopt_long(argc, argv, "a:df:hIlp:vV", OPTIONS, NULL))
-           != -1)
+    while ((c = getopt_long(argc, argv, "a:df:hIlp:vV", OPTIONS, NULL)) != -1)
     {
         switch (c)
         {
-        case 'a':
-            SetBindInterface(optarg);
-            break;
-        case 'd':
-            LogSetGlobalLevel(LOG_LEVEL_DEBUG);
-            break;
-        case 'f':
-            config->file = xstrdup(optarg);
-            break;
-        case 'h':
-            CFTestD_Help();
-            exit(EXIT_SUCCESS);
-        case 'I':
-            LogSetGlobalLevel(LOG_LEVEL_INFO);
-            break;
+        case 'a': SetBindInterface(optarg); break;
+        case 'd': LogSetGlobalLevel(LOG_LEVEL_DEBUG); break;
+        case 'f': config->file = xstrdup(optarg); break;
+        case 'h': CFTestD_Help(); exit(EXIT_SUCCESS);
+        case 'I': LogSetGlobalLevel(LOG_LEVEL_INFO); break;
```

Comment:
I like the original multi-line version better.

Reply:
I agree, I can change the config for this, I think almost all of our codebase has these on separate lines.